src/scripts/towns/Town.ts

```diff
@@ -2136,4 +2136,280 @@ TownList['Resolution Cave'] = new DungeonTown(
+    'There are plenty of unique, powerful ones there, too!',
+]);
+
+
+//Galar towns
+
+TownList['Postwick'] = new Town(
+    'Postwick',
+    GameConstants.Region.galar,
+    {
+        shop: PostwickShop,
+        npcs: [Mom],
+    }
+);
+TownList['Wedgehurst'] = new Town(
+    'Wedgehurst',
+    GameConstants.Region.galar,
+    {
+        shop: WedgehurstShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 1)],
+        npcs: [TrainStationGuy],
+    }
+);
+TownList['Motostoke'] = new Town(
+    'Motostoke',
+    GameConstants.Region.galar,
+    {
+        shop: MotostokeShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 6)],
+    }
+);
+TownList['Turffield'] = new Town(
+    'Turffield',
+    GameConstants.Region.galar,
+    {
+        shop: TurffieldShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 11)],
+    }
+);
+TownList['Hulbury'] = new Town(
+    'Hulbury',
+    GameConstants.Region.galar,
+    {
+        shop: HulburyShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 12)],
+    }
+);
+TownList['Stow-on-Side'] = new Town(
+    'Stow-on-Side',
+    GameConstants.Region.galar,
+    {
+        shop: StowonSideShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 15)],
+    }
+);
+TownList['Ballonlea'] = new Town(
+    'Ballonlea',
+    GameConstants.Region.galar,
+    {
+        requirements: [new ClearDungeonRequirement(1, GameConstants.getDungeonIndex('Glimwood Tangle'))],
+    }
+);
+TownList['Hammerlocke'] = new Town(
+    'Hammerlocke',
+    GameConstants.Region.galar,
+    {
+        shop: HammerlockeShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 14)],
+    }
+);
+TownList['Circhester'] = new Town(
+    'Circhester',
+    GameConstants.Region.galar,
+    {
+        shop: CirchesterShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 18)],
+    }
+);
+TownList['Spikemuth'] = new Town(
+    'Spikemuth',
+    GameConstants.Region.galar,
+    {
+        shop: SpikemuthShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 22)],
+    }
+);
+TownList['Wyndon'] = new Town(
+    'Wyndon',
+    GameConstants.Region.galar,
+    {
+        shop: WyndonShop,
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.galar, 24)],
+        dungeon: dungeonList['Rose Tower'],
+    }
+);
+TownList['Wyndon Stadium'] = new Town(
+    'Wyndon Stadium',
+    GameConstants.Region.galar,
+    {
+        requirements: [new ClearDungeonRequirement(1, GameConstants.getDungeonIndex('Rose Tower'))],
+    }
+);
+//Isle of Armor towns
+
+TownList['Master Dojo'] = new Town(
+    'Master Dojo',
+    GameConstants.Region.armor,
+    {
+        requirements: [new RouteKillRequirement(10, GameConstants.Region.armor, 1)],
+        dungeon: dungeonList['Master Dojo'],
```

Comment:
The dungeon and town can't both have the same name, One of them will have to be renamed otherwise it seems to cause issues.

Reply:
Or just convert this to a DungeonTown